docs/data_preparation.rst

```diff
@@ -641,20 +641,31 @@ Other Data (multivec)
 
 Multivec files are datatype agnostic. For use with generic data, create a
 `segments` file containing the maximum value for each segment. A segment is an
-arbitrary set of discontinuous blocks that the data is partitioned into. If the
+arbitrary set of discontinuous blocks that the data is partitioned into. In the
+case of genomics data, segments correspond to chromosomes. If the
 data has no natural grouping, one segment can be used which contains the
 maximum x value in the dataset:
 
 .. code-block:: bash
 
     segment1    20000
+    segment2    40000
 
-The individual datapoints should then be formatted as follows:
+Data will be displayed as if the segments were laid out end to end:: 
 
 .. code-block:: bash
 
-    segment_name    start   end value
-    segment1    10  100 5
+    |---------------|------------------------------|
+         segment1               segment2
+
+The individual datapoints should then be formatted as in the block below. Each
+row in this file corresponds to a column in the displayed matrix. Each ``value``
+is a row in the matrix. 
```

Comment:
What is this matrix you are referring to? Are you saying that 

```
segment1           10  100       5      15       20
```

is translated into a stacked bar that covers the range [10-100] and has 3 segments defined by the 3 value columns?

I am still confused how this row defining `segment1` with the list above where it says `segment1 20000 `